Topics/Development_Process/QA_testing.md

```diff
@@ -46,11 +67,47 @@ There are many reasons why software testing is important. This blog [7 reasons w
+
+1. Define the area you want to cover from the test scenario.
+2. Ensure the test case is easy for testers to understand and execute.
+3. Understand and apply relevant test designs.
+4. Use a unique test case ID.
+5. Use the requirements traceability matrix in testing for visibility.
+6. Include a clear description in each test.
+7. Add proper preconditions and postconditions.
+8. Specify the exact expected result. 
+9. Utilize suitable testing techniques. 
+10. Get your test plan peer-reviewed before moving forward.
+
+The article also included a list of popular test case management tools. You may want to have some experience with one or two of them if you are about to engage in writing tests for a large formal project: 
+
+1. [JIRA](https://www.atlassian.com/software/jira)
+2. [Juno.one](https://www.juno.one/)
+3. [Klaros Test Management](https://www.klaros-testmanagement.com/en/)
+4. [QACoverage](https://www.qacoverage.com/)
+5. [Qase](https://qase.io/)
+6. [SPIRATEST](https://www.inflectra.com/SpiraTest/) by Inflectra
+7. [TestFLO](https://marketplace.atlassian.com/apps/1211393/testflo-test-management-for-jira) for JIRA
+8. [Testpad](https://testpad.com/)
+9. [XQual](https://www.xqual.com/)
+10. [Xray](https://www.getxray.app/)
+11. [Zephyr Scale](https://marketplace.atlassian.com/apps/1213259/zephyr-scale-test-management-for-jira) for JIRA
+12. [Zephyr Squad](https://marketplace.atlassian.com/apps/1014681/zephyr-squad-test-management-for-jira) for JIRA
+
+There are some important takeaways from this article: 
+
+- Test scenarios v.s. test cases: test scenarios often refer to the broader pictures, describing how the user expects to complete with the software. Test cases refer to the details in the test scenarios, such as a specific way that the user may attempt to achieve the goal.
+- Who should write the QA test cases: It's best to have people not involved in coding the product write the test cases because it might bring in fresh perspectives.
+
+If you need more help on how to write a test case, here is a [short blog](https://blog.testlodge.com/how-to-write-test-cases-for-software-with-sample/) that's relatively easy to understand. This blog provides visual examples of what a test case should look like, and a [short video](https://www.youtube.com/watch?v=khGa1Rdzd2A) showing step-by-step how to write a test case.
 
 ## Automated testing
 
-[This article](./Automated_Testing.md) provides a nice overview of automated testing, which is an important tool in QA testing.
+Automated testing is an important part of QA testing, that involves engineers writing test scripts, and running them to get test results. Since the test scripts can be reused, it's extremely suitable for conducting tests repeatedly over many iterations of a product. Sometimes, there is a dedicated QA team that writes automated tests. Sometimes, the software development team will do the work.
+
```

Comment:
it is better to make a list to illustrate why the automated testing is important. and readable?